docs/source/traitsui_user_manual/factories_advanced_extra.rst

```diff
@@ -553,7 +553,8 @@ TabularEditor()
 :Optional parameters:
     *activated, clicked, column_clicked, dclicked, drag_move, editable,*
     *horizontal_lines, images, multi_select, operations, right_clicked,*
-    *right_dclicked, selected, selected_row, show_titles, vertical_lines*
+    *right_dclicked, scroll_to_position_hint, selected, selected_row,
+    *show_titles, vertical_lines*
```

Comment:
i don't think there is an issue regd the discussion we had about pointing to the API docs here instead of manually listing the traits here

Reply:
I've opened #1591